composer.json

```
@@ -31,7 +31,7 @@
     "pimcore/admin-ui-classic-bundle": "^1.0 || ^2.0",
     "pimcore/object-merger": "^4.0",
     "pimcore/pimcore": "^11.2 || ^12.0",
-    "pimcore/search-query-parser": "^1.3",
+    "pimcore/search-query-parser": "1.x-dev",
```

```suggestion
    "pimcore/search-query-parser": "^2.0",
```
apply and merge when tagging that bundle